src/xdocs/config_filters.xml

```diff
@@ -1011,6 +1011,419 @@ public class InputTest {
+          inside FileOne and FileTwo files:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="File(One|Two)\.java"/&gt;
+  &lt;property name="checks" value="MethodName"/&gt;
+  &lt;property name="query" value="(/CLASS_DEF[@text='FileOne']/OBJBLOCK/
+            METHOD_DEF[@text='MyMethod']/IDENT)|
+            (/CLASS_DEF[@text='FileTwo']/OBJBLOCK/METHOD_DEF[@text='MyMethod']/IDENT)"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>Code example:</p>
+        <pre>
+public class FileOne {
+  public void MyMethod() {} // OK
+}
+
+public class FileTwo {
+  public void MyMethod() {} // OK
+}
+
+public class FileThree {
+  public void MyMethod() {} // violation, name 'MyMethod'
+                            // must match pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
+}
+        </pre>
+        <p>
+          To suppress MethodName check for method names matched pattern 'MyMethod[0-9]':
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="checks" value="MethodName"/&gt;
+  &lt;property name="message" value="MyMethod[0-9]"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>Code Example:</p>
+        <pre>
+public class FileOne {
+  public void MyMethod1() {} // OK
+  public void MyMethod2() {} // OK
+  public void MyMethodA() {} // violation, name 'MyMethodA' must
+                             // match pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
+}
+        </pre>
+        <p>
+          To suppress checks being specified by id property:
+        </p>
+        <pre>
+&lt;module name="MethodName"&gt;
+  &lt;property name="id" value="MethodName1"/&gt;
+  &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
+&lt;module/&gt;
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileOne.java"/&gt;
+  &lt;property name="id" value="MethodName1"/&gt;
+&lt;module/&gt;
+        </pre>
+        <p>Code example:</p>
+        <pre>
+public class FileOne {
+  public void MyMethod() {} // OK
+}
+public class FileTwo {
+  public void MyMethod() {} // violation,  name 'MyMethod' must
+                            //match pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
+}
+        </pre>
+        <p>
+          To suppress checks for all package definitions:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter&gt;
+  &lt;property name="checks" value="PackageName"/&gt;
+  &lt;property name="query" query="/PACKAGE_DEF[@text='File']/IDENT"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>Code example:</p>
+        <pre>
+package File; // OK
+
+public class FileOne {
+  public void sayHelloWorld() {
+    int num = 10;
+
+    if (num == 1 || num == 2) {
+      // ...
+    } else if (num == 3 || num == 4) {
+      // ...
+    } else if (num == 5 || num == 6) {
+      // ...
+    } else if (num == 7 || num == 8) {
+      // ...
+    } else if (num == 9 || num == 10) {
+      // ...
+    } else {
```

Comment:
remove extra code.
please keep only code that is required for example. FileOne should be empty class.

Reply:
Done.